locales-pending/onboarding.ftl

```diff
@@ -19,9 +19,9 @@ onboarding-get-started-how-it-works-dialog-step1-content = {
         *[other] With just a few key pieces of information, we’ll search for you in all known data breaches and { $dataBrokerTotalCount } major data broker sites. All users get the first scan free.
 }
 onboarding-get-started-how-it-works-dialog-step2-heading = Remove your personal info
-onboarding-get-started-how-it-works-dialog-step2-content = We’ll show you what personal info each data broker site is selling. You’ll need to contact each data broker to request removal. Removal typically takes 7-10 business days, but some can take longer. Upgrade to { -brand-premium } and we’ll handle this for you.
+onboarding-get-started-how-it-works-dialog-step2-content = We’ll show you what personal info each data broker site is selling. You’ll need to contact each data broker to request removal. Removal typically takes 7-10 business days, but some can take longer. Subscribe to { -brand-monitor-plus } and we’ll handle this for you.
 onboarding-get-started-how-it-works-dialog-step3-heading = Resolve data breaches
-onboarding-get-started-how-it-works-dialog-step3-content = We’ll give you clear steps on how to resolve each data breach. They’ll require access to your accounts, so you’ll need to resolve them manually even if you upgrade to { -brand-premium }.
+onboarding-get-started-how-it-works-dialog-step3-content = They’ll require access to your accounts, so you’ll need to resolve them manually even if you upgrade to { -brand-monitor-plus }.
```

Comment:
Looks like this might be one of the only remaining instances of "**upgrade** to Plus" versus "**subscribe** to Plus":
```sh
git grep -in "uPgRadE To" -- ":(exclude)docs/**" ":(exclude)*.yaml" | cat

locales-pending/onboarding.ftl:24:onboarding-get-started-how-it-works-dialog-step3-content = They’ll require access to your accounts, so you’ll need to resolve them manually even if you upgrade to { -brand-monitor-plus }.
src/e2e/pages/dashBoardPage.ts:52:      name: "Upgrade To Premium",
```

Reply:
Thanks, this looks correct per Figma so I'll ask there.

Reply:
@mozrokafor on the e2e test, it didn't seem to break when I changed the string, is this a part of the test that runs on PR? I'll go ahead and update it but curious if you know why.